packages/provider/package.json

```diff
@@ -34,7 +34,11 @@
     "@types/node": "20.17.24",
     "@vercel/ai-tsconfig": "workspace:*",
     "tsup": "^8",
-    "typescript": "5.8.3"
+    "typescript": "5.8.3",
+    "zod": "3.23.8"
+  },
+  "peerDependencies": {
+    "zod": "^3.23.8"
```

Comment:
Please remove Zod. The provider pkg should not have this dependency.

Reply:
will do 🙏

Reply:
done 👍